.github/workflows/build-container.yml

```diff
@@ -0,0 +1,74 @@
+name: Build and Push Docker image
+
+on:
+  push:
+    branches: [ master ]
+  repository_dispatch:
+    types:
+      - dispatch-build
+  workflow_dispatch:
+      
+jobs:
+  make-date-tag:
+    runs-on: ubuntu-latest
+    if: startsWith(github.repository, 'opensciencegrid/')
+    outputs:
+      dtag: ${{ steps.mkdatetag.outputs.dtag }}
+    steps:
+    - name: make date tag
+      id: mkdatetag
+      run: echo "::set-output name=dtag::$(date +%Y%m%d-%H%M)"
+
+  build:
+    runs-on: ubuntu-latest
+    needs: [make-date-tag]
+    if: startsWith(github.repository, 'opensciencegrid/')
+    strategy:
+      fail-fast: False
+      matrix:
+        repo: ['latest']
```

Comment:
```suggestion
```

A matrix of zero is a little silly

Reply:
it's a matrix of one though ...........

Reply:
Ah yeah, I meant one!

Reply:
>matrix of zero

in college, for a brief revelatory period, i seemed to discover the deepest of mathematical truths:

the zero-by-zero matrix sits undetected at the deepest foundations of all mathematics.

and yet, vacuously so.

In the zero-by-zero dimension, every matrix is the Zero matrix _and_ every matrix is the One matrix, and that matrix, which is every matrix, is the Determinant.

Call it silly if you like, but mind equalled blown, i'm telling you man.